### PR TITLE
Added C# (.NET) definition with PostgreSQL container

### PR DESCRIPTION
**Pull request by Berkays**

### CODE REVIEW

**Comment by dnastrain** [comment]
Very cool to see, thank you for adding this!  
FYI, we were informed that we should remove SQLTools from other containers definitions due to the actively tracked issue: 
https://github.com/microsoft/vscode-dev-containers/issues/1253 -- "SQLTools extension went "archived", should remove references to it in definitions"

**Comment by Berkays** [comment]
@dnastrain I will edit my PR to remove the extension. Should I replace it with `
ms-ossdata.vscode-postgresql`? I personally add a network configuration to let my pgAdmin container access to my setups. Will it be better to include a such management tool in the compose instead of an extension?

**Comment by dnastrain** [comment]
> @dnastrain I will edit my PR to remove the extension. Should I replace it with ` ms-ossdata.vscode-postgresql`? I personally add a network configuration to let my pgAdmin container access to my setups. Will it be better to include a such management tool in the compose instead of an extension?

hi @Chuxel, would you please share your thoughts on the question above?

**Comment by Chuxel** [comment]
@dnastrain @Berkays Ack - Lost track here. The `ms-ossdata.vscode-postgresql` extension has not been updated in many years, so I do not think we should include it.

File: containers/dotnet-postgres/.devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "name": "C# (.NET) and PostgreSQL",
```
Comment (Chuxel) [inline containers/dotnet-postgres/.devcontainer/devcontainer.json:2]: We've been including this in community contributed definitions.
```suggestion
    "name": "C# (.NET) and PostgreSQL (Community)",
```

File: containers/dotnet-postgres/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
+# C# (.NET) and MS SQL
```
Comment (Chuxel) [inline containers/dotnet-postgres/README.md:1]: Looks like the title here is incorrect.
```suggestion
# C# (.NET) and PostgreSQL (Community)
```

File: containers/dotnet-postgres/README.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+## Summary
+
+*Develop C# and .NET Core based applications. Includes all needed SDKs, extensions, dependencies and a PostgreSQL container for parallel database development. Adds an additional PostgreSQL container to the C# (.NET Core) container definition.*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | The VS Code Team, [Berkays](https://github.com/Berkays)  |
```
Comment (Chuxel) [inline containers/dotnet-postgres/README.md:9]: I'm assuming you are okay with helping to maintain this definition?  If so, you can add yourself as the contributor.
```suggestion
| *Contributors* | [Berkays](https://github.com/Berkays)  |
```

File: containers/dotnet-postgres/README.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+| *Categories* | Core, Languages |
+| *Definition type* | Docker Compose |
+| *Published image architecture(s)* | x86-64 |
+| *Available image variants* | 3.1-focal, 5.0-focal, 6.0-focal |
```
Comment (Chuxel) [inline containers/dotnet-postgres/README.md:13]: Does this work with the `-bullseye` definitions as well?  If so, you can change this as follows so you don't have to keep updating as new image variations appear:
```suggestion
| *Available image variants* | [See C# (.NET) definition](../dotnet). |
```

File: containers/dotnet-postgres/.devcontainer/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# [Choice] .NET version: 6.0, 5.0, 3.1
```
Comment (Chuxel) [inline containers/dotnet-postgres/.devcontainer/Dockerfile:1]: Assuming that this works with the `-bullseye` variants of the images, you can update this to:
```suggestion
# [Choice] .NET version: 6.0, 5.0, 3.1, 6.0-bullseye, 5.0-bullseye, 3.1-bullseye, 6.0-focal, 5.0-focal, 3.1-focal
```

File: containers/dotnet-postgres/.devcontainer/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+ARG VARIANT="6.0"
+FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}-focal
```
Comment (Chuxel) [inline containers/dotnet-postgres/.devcontainer/Dockerfile:3]: Similarly, you can update this as follows if bullseye works:
```suggestion
FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}
```

**Review by Chuxel — APPROVED** [review]
LGTM!

**Pull request merged**